tests/drivers/uart/uart_elementary/prj.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_SERIAL=y
+CONFIG_ZTEST=y
+CONFIG_TEST_USERSPACE=y
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_UART_USE_RUNTIME_CONFIGURE=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_UART_NRFX_UARTE_LEGACY_SHIM=n
```

Comment:
it's nordic specific Kconfig and test is generic so it shouldn't be here. It's the default option for nrf54l/h so it's only have to be forced for nrf53 (maybe board specific conf file?).